openstack/messaging/v2/messages/results.go

```diff
@@ -12,6 +15,22 @@ type CreateResult struct {
 	gophercloud.Result
 }
 
+// MessagePage contains a single page of all clusters from a ListDetails call.
+type MessagePage struct {
+	pagination.LinkedPageBase
+}
+
+// Message represents a message on a queue.
+type Message struct {
+	Body     map[string]interface{} `json:"body"`
+	Age      int                    `json:"age"`
+	Href     string                 `json:"href"`
+	ID       string                 `json:"id"`
+	TTL      int                    `json:"ttl"`
+	Delay    int                    `json:"delay"`
```

Comment:
I'm not able to find where "delay" is defined as a field. It looks like all of the other fields are correct:

https://github.com/openstack/zaqar/blob/a8215f72f960aeb22f3e3e30abf79b74617def4f/zaqar/api/v2/response.py#L32-L60

Can you verify this?

Reply:
You are correct. Sorry about that.